docs/UserGuide.md

```diff
@@ -324,6 +325,35 @@ Examples:
 
 ![result for 'edit event'](images/editEventResult.png)
 
+--------------------------------------------------------------------------------------------------------------------
+
+### Sorting the patient list : `sort`
+
+Sorts the patient list in the address book by specified attribute.
+
+Format: `sort [ATTRIBUTE]`
+
+* Sorts all the added patients in the patient list by specified `ATTRIBUTE`.
+* Supported `ATTRIBUTE`s include `n` for patient name and `p` for patient's preferred name.
+* There can only be 1 or 0 `ATTRIBUTE`.
+* If no `ATTRIBUTE` is specified, the `sort` command will wort the patient list by patient's name by default.
```

Comment:
~wort~ sort

Reply:
Good Catch! will proceed to fix the spelling error!